HotelManagement/src/main/java/com/deltasource/hotelmanagement/core/Manager.java

```diff
@@ -27,6 +42,7 @@ public String getName() {
 
 	/**
 	 * Sets the name of the manager.
+	 *
```

Comment:
Don't need this. Only constructor should be able to set the name. Too much hassle to change the name of a person.